lib/new_relic/harvest/collector/custom_event/harvester.ex

```diff
@@ -51,7 +51,7 @@ defmodule NewRelic.Harvest.Collector.CustomEvent.Harvester do
 
   def complete(harvester) do
     Task.Supervisor.start_child(Collector.CustomEvent.TaskSupervisor, fn ->
-      GenServer.call(harvester, :send_harvest)
+      GenServer.call(harvester, :send_harvest, 15_000)
```

Comment:
Is it worth extracting this into a module function so that all collectors utilize the same timeout?